blackbox/suites/community/pjdfstest/module.py

```
@@ -30,19 +34,24 @@ def run(self, ctx: Context) -> dict[str, Any]:
-            # the harness itself stays unprivileged. If sudo is unavailable or
-            # not passwordless, fail hard rather than running a degraded suite.
-            prove_cmd = ["prove", "--recurse", "--verbose", *test_args]
+            # etc.) that require root. When not root, elevate via passwordless
+            # sudo so the harness itself stays unprivileged. If sudo is
+            # unavailable or not passwordless, fail hard rather than running a
+            # degraded suite.
+            prove_cmd = [prove_bin, "--recurse", "--verbose", *test_args]
             if not ctx.capabilities.get("is_root"):
                 if not shutil.which("sudo"):
                     raise BlackboxError("pjdfstest requires root or passwordless sudo; sudo not found")
                 probe = subprocess.run(["sudo", "-n", "true"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=False)
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)